config/tcs_config.toml

```diff
@@ -13,10 +13,8 @@
 # limitations under the License.
 
 [KvStorage]
-StoragePath = "config/Kv_Shared_tmp"
-StorageSize = "1 TB"
 # the remote version is of higher priority if enabled
-#remote_url = "http://localhost:9090"
+remote_url = "http://127.0.0.1:9090"
```

Comment:
Use `localhost` instead of hard-coding to `127.0.0.1`. This will allow it to work with IPv6.

Reply:
Agree and changed to localhost